src/theme.ts

```diff
@@ -1,4 +1,4 @@
-export const COLORS: {[name: string]: Theme} = {
+export const COLORS: { [name: string]: Theme } = {
```

Comment:
Can you remove the spaces?

Reply:
Yeah. sure, but why??

Reply:
I misunderstood. There is no problem because it conforms to deno fmt.

Reply:
ok